core/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -732,6 +732,16 @@ public void testSizeFormatInvalidArgumentSize()
           .eval(bindings);
   }
 
+  @Test
+  public void testSafeDivide()
+  {
+    // happy path maths
+    assertExpr("safe_divide(3, 1)", 3L);
+    assertExpr("safe_divide(4.5, 2)", 2.25);
+    assertExpr("safe_divide(3, 0)", 0L);
+    assertExpr("safe_divide(3.7, 0.0)", 0L);
```

Comment:
if other suggestion is applied:

```suggestion
    assertExpr("safe_divide(3, 0)", NullHandling.defaultLongValue());
    assertExpr("safe_divide(3.7, 0.0)", NullHandling.defaultDoubleValue());
```

Reply:
Done